indexer/src/utils/filter.test.ts

```diff
@@ -112,37 +110,34 @@ Deno.test("isKakarotTransaction: no `to` field in calldata", () => {
   assertFalse(failed);
 });
 
-Deno.test("isKakarotTransaction: `to` address not matching KAKAROT_ADDRESS", () => {
-  const starknetTxCalldata: `0x${string}`[] = [
-    "0x1",
-    "0x2",
-  ];
-  const transaction: Transaction = {
-    invokeV1: {
-      senderAddress: "0x01",
-      calldata: starknetTxCalldata
-    },
-    meta: {
-      hash: "0x02",
-      maxFee: "0x02",
-      nonce: "0x02",
-      signature: ["0x2"],
-      version: "1",
-    },
-  };
-  const failed = isKakarotTransaction(transaction);
-  assertEquals(failed, false);
-});
+Deno.test(
+  "isKakarotTransaction: `to` address not matching KAKAROT_ADDRESS",
+  () => {
+    const starknetTxCalldata: `0x${string}`[] = ["0x1", "0x2"];
+    const transaction: Transaction = {
+      invokeV1: {
+        senderAddress: "0x01",
+        calldata: starknetTxCalldata,
+      },
+      meta: {
+        hash: "0x02",
+        maxFee: "0x02",
+        nonce: "0x02",
+        signature: ["0x2"],
+        version: "1",
+      },
+    };
+    const failed = isKakarotTransaction(transaction);
+    assertEquals(failed, false);
+  },
+);
```

Comment:
we should add a linter in CI, like `deno fmt --check indexer/`, can make a good first issue

Reply:
Related https://github.com/kkrt-labs/kakarot-rpc/issues/1147